code/src/main/kotlin/com/expediagroup/sdk/core/model/exception/service/ExpediaGroupServiceDefaultErrorException.kt

```diff
@@ -15,4 +15,5 @@
  */
 package com.expediagroup.sdk.core.model.exception.service
 
-class ExpediaGroupServiceDefaultErrorException(code: Int, override val errorObject: String, transactionId: String?) : ExpediaGroupApiException(code, errorObject, transactionId)
+class ExpediaGroupServiceDefaultErrorException(code: Int, override val errorObject: String, transactionId: String?) :
```

Comment:
I know this is here from the existing core, but why do we use it? The name is confusing. Can we replace it with a more defined name(s)?

Reply:
Deleted class since it is unused.